modules/apps/frontend-token/test.properties

```diff
@@ -0,0 +1,18 @@
+##
+## DO NOT EDIT THIS FILE.
+##
+## To update the properties of this file, create a separate properties file
+## named "test.${user.name}.properties" with the properties to overwrite.
+##
+
+##
+## Test Batch
+##
+
+    #
+    # Relevant
+    #
+
+    test.batch.run.property.query[functional-tomcat*-mysql*-jdk8][relevant]=\
+        (portal.acceptance == true) AND \
+        (testray.component.names ~ "Style Books")
```

Comment:
use:
testray.main.component.name

Take note of what properties are set in the StyleBooks.testcase.

Reply:
use:

(testray.component.name == "Frontend Token") OR \
 (\
(portal.acceptance == true) AND \
(testray.main.component.name ~ "Style Books")
)